celery/backends/cassandra.py

```diff
@@ -8,7 +8,7 @@
 from celery.exceptions import ImproperlyConfigured
 from celery.utils.log import get_logger
 
-from .base import BaseBackend
+from .base import StructuredStoreBackend
```

Comment:
How do I test that the Cassandra back-end actually works?
I really don't have time to do a manual test and the unit tests don't actually test with a live Cassandra database unlike the SQLAlchemy back-end.